### ISSUE TITLE
[ANALYTICS][PFR]Add a "Mixed activity" type for an activity having a combination of different activity types

### ISSUE THREAD

**Issue by sirinebenabbes** [Meeds-io/meeds#471]
**Description**

When posting an activity that has a link, attached files, etc. at the same time, we can't trace all of these activity types in the analytics data i.e **ACTIVITY_LINK** and **files:spaces** since only **one activity type** could be traced. 
This impacts the analytics results of activity type filter reports.

**Suggestion**
Add a new activity type **Mixed activity** with which we can list the combination of the different activity types of one post/activity

**Comment by sirinebenabbes** [comment]
@Imenmakni @Samuel-Renault-eXo WDYT ?

**Comment by srenault-meeds** [comment]
Yes indeed, that can be a solution. What is the ETA for this please? We might need to discuss it in the product committee.
@margondicco
